controlplane/pkg/common/context.go

```diff
@@ -270,3 +272,20 @@ func WorkspaceName(ctx context.Context) string {
 	}
 	return value.(string)
 }
+
+func WithSecurityContext(parent context.Context, sc security.Context) context.Context {
+	if parent == nil {
+		parent = context.Background()
+	}
+	return context.WithValue(parent, securityContextKey, sc)
+}
+
+func SecurityContext(ctx context.Context) security.Context {
+	value := ctx.Value(securityContextKey)
+	if value == nil {
+		logrus.Info("IT IS NIL")
```

Comment:
:)

Reply:
I know that I invited everyone to review it regardless PR is draft, but now it is in draft for real, because I adapt it to new unified api and SDK like refactoring. I just didn't expect that someone starts review it now :)

Reply:
got it. Let me know when it's ready for review.